compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -144,6 +144,23 @@ def PadTensorToSubTensorInsert :
   let constructor = "mlir::iree_compiler::IREE::Flow::createPadTensorToSubTensorInsertPass()";
 }
 
+def PrintDispatchGraph : Pass<"iree-flow-print-dispatch-graph"> {
```

Comment:
name this DumpDispatchGraph
(keeping the naming between flags, files, and passes is important for managing complexity when there are dozens/hundreds of each)